case-server/src/main/java/com/powsybl/caseserver/repository/CaseMetadataEntity.java

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright (c) 2022, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.caseserver.repository;
+
+import java.time.LocalDateTime;
+import java.util.UUID;
+
+import lombok.*;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+/**
+ * @author Abdelsalem Hedhili <abdelsalem.hedhili at rte-france.com>
+ */
+@NoArgsConstructor
+@AllArgsConstructor
+@Getter
+@Setter
+@Entity
+@Table(name = "caseMetadata")
+public class CaseMetadataEntity {
+
+    @Id
+    @Column(name = "id")
```

Comment:
Autogenerate ?

Reply:
I think it's not useful here. I need the id to create the repository in which the case will be stored. So it's better if I generate it myself as I need it before I create this entity.